packages/v4/src/content/design-guidelines/components/tabs/tabs.md

```diff
@@ -2,4 +2,64 @@
+
+- [Default tabs](#default-tabs) (recommended)
+- [Boxed tabs](boxed-tabs)
+- [Vertical tabs](#vertical-tabs)
+
+You can use as many tabs as you want in a page, although it is recommended you keep the total number of tabs to a minimum for a better user experience.If you have more tabs than fit on the page, you may use a [tab overflow](/components/tabs#default-overflow) which would allow a user to horizontal scroll through your tabs. The uses of both default and boxed stylings are flexible, so use either according to your needs and preferences. 
+
+### Default tabs
+
+Default tabs do not have any borders, other than the bottom line to distinguish between a selected tab, a hovered over tab, and a non-active tab. These tabs are most commonly used as top page header tabs or nested within components.
+
+<img src="./img/default-tabs.png" alt="Default tabs ex" width="1500"/>
+
+Default tabs can be formatted as standard or filled.  Filled tabs are placed side by side and span the whole width of the page. They can be used in screens with narrower widths, such as a drawer within a [primary detail view](/demos/primary-detail). When in doubt, stick with the standard styled tabs. 
+
+<img src="./img/default-vs-filled-tabs.png" alt="Standard and filled tabs ex" width="1500"/>
+
+1. Standard styled tabs
+2. Filled styled tabs
+
+### Boxed tabs
+
+Box tabs are functionally the same as default tabs, but they are outlined by a box to emphasize the area that the tab spans. Boxed tabs can be used interchangeably with default tabs, but we recommend using the default tabs over boxed tabs. Boxed tabs can be used in combination with default tabs to create more hierarchy on the page.
+
+<img src="./img/boxed-tabs.png" alt="Boxed tabs ex" width="1500"/>
+
+### Vertical tabs
+
+Vertical tabs are placed on the left hand side of a page or container. 
+
+<img src="./img/vertical-tabs.png" alt="Vertical tabs ex" width="1500"/>
+
+## Hierarchy
+
+Both boxed and default tabs can help create hierarchy on the page. We recommend using default tabs at all times, although mixing the two styles can separate sections of the page more distinctly. When using many levels of default tabs, make sure to incorporate the following types.
+
+Two tab options provide hierarchy within a page:
+- [Primary tabs](#primary-tabs)
+- [Secondary tabs](#secondary-tabs)
+
+### Primary tabs
+
+Use these only for top page header tabs and vertical tabs. Primary tabs use 16px text. 
+
+<img src="./img/primary-tabs.png" alt="Primary tabs ex" width="1500"/>
+
+### Secondary tabs
+
+Use these for any use case that differs from primary tabs. Secondary tabs use 14px text. They are especially useful when another set of tabs are needed after primary as well as when tabs are needed within a component.
+
+<img src="./img/secondary-1-tabs.png" alt="Secondary tabs ex" width="1500"/>
+
+1. Primary tabs
+2. Secondary tabs
+
+<img src="./img/secondary-2-tabs.png" alt="Secondary tabs ex" width="1500"/>
```

Comment:
Are you going to say anything about the last example? Seems to be just hanging there.

Reply:
Agreed^ would maybe move the image up, so that both images are together, and change the little call out to have a "2" in it to show that it's a secondary tab as well.

Although now i'm confused about primary and secondary tabs. I thought secondary tabs should only be used when there are 2 levels of tabs, and in this modal example there's only one level so why would it be a secondary tab?

Reply:
Yes added some text! @mcarrano @mmenestr